### ISSUE TITLE
Support split view

### ISSUE THREAD

**Issue by jzaefferer** [scottgonzalez/pretty-diff#13]
Like the new one GitHub has.

**Comment by scottgonzalez** [comment]
Just pushed some updates. I've been thinking about changing the handling for the file names for a while. I'll tackle that separately though. Let me know if you find any problems with it now.

**Comment by jzaefferer** [comment]
I ran `npm install -g scottgonzalez/pretty-diff#split` to install this again.

For [this commit](https://github.com/jzaefferer/qunit/commit/6b014564874928b7574729849cbc47eb05a23487), I think the output is worse then before. The last change doesn't line up. That also happens for the other commits I mentioned before:

![...](http://bassistance.de/i/1396be.png)

**Comment by scottgonzalez** [comment]
Yeah, I broke the detection of the first line in the lookahead, so everything was off. I also just realized why GitHub is using tables for split view; it's needed to handle one side of the diff having a long line which wraps when the other side doesn't. I'll get that updated soon too.

**Comment by jzaefferer** [comment]
Looks better now. Let me know when you got this tabled, and I'll test again.

Regarding configuration: I like GitHub's model of remember the choice and using that for all diffs until I pick the other view. Maybe you can do the same on the CLI? Once I pick split view using an option, make that the default until I switch again. Like `--split` and `--unified`.

**Comment by tauren** [comment]
This is great stuff, nice work! 

I have a use for both your `split` branch and your `word-diff` branch. Will they eventually be usable together and published to NPM? I can make due for now, just interested in your thoughts on the project's roadmap.

**Comment by scottgonzalez** [comment]
> Will they eventually be usable together and published to NPM?

I'm not sure that word-diff will ever be usable. If it is, then they'll work together.

**Comment by jzaefferer** [comment]
Btw. I've been using the split view since January or so, haven't noticed any other issues.

**Comment by jzaefferer** [comment]
I don't know where my previous comment came from, I haven't actually used the split view locally...
